backend/src/test/java/com/soen341/instagram/service/impl/PictureServiceTest.java

```diff
@@ -0,0 +1,124 @@
+package com.soen341.instagram.service.impl;
```

Comment:
Are any of these tests expecting some sort of error response or exception thrown? I'm not too familiar with writing tests but I think from @nasaku898 there is a way to clearly show this and also make it clear in the function name.

Reply:
You're right. Fixed!